### PR TITLE
Quality of life changes

### PR DESCRIPTION
**Pull request by mallport**
This PR was created in order to separate the smaller refactoring changes from the functional changes in depseudo/repseudo, which will come in a future PR.

**Changes**
- Rename files to drop the 'builder'-prefix, as it will soon become the sole interface. I.e. `builder_pseudo.py` -> `pseudo.py`
- Update README to match the new interface. Examples now also use Polars instead of Pandas to show functionality, as it is the preferred dataframe to use with this library.
- Proper support for both defining custom, predefined keys _and_ keysets
- Rename the Builder intermediate-classes. Previously, we had:

```python
PseudoData
    .from_pandas(df)
    .on_fields("fnr")
    .with_default_encryption()
    .pseudonymize()
```

..while now, it is:

```python
Pseudonymize
    .from_pandas(df)
    .on_fields("fnr")
    .with_default_encryption()
    .run()
```

This will make it more ergonomic as we introduce depseudo/repseudo, which will need their own builders, and can be called `Depseudonymize` and `Repseudonymize`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,76 +32,79 @@ Pseudonymize, repseudonymize and depseudonymize data on Dapla.
...
     .to_polars()                                   # Get the result as a polars dataframe
 )
 
 # Example: Multiple fields default encryption (DAEAD)
 result_df = (
-    PseudoData.from_pandas(df)                     # Specify what dataframe to use
+    Pseudonymize.from_polars(df)                   # Specify what dataframe to use
     .on_fields("fornavn", "etternavn")             # Select multiple fields to pseudonymize
+    .with_default_encryption()                     # Select the pseudonymization algorithm to apply
     .pseudonymize()                                # Apply pseudonymization to the selected fields
```
Comment (bjornandre) [inline README.md:56]:
```suggestion
    .run()                                # Apply pseudonymization to the selected fields
```

File: README.md [hunk 2]
```diff
@@ -32,76 +32,79 @@ Pseudonymize, repseudonymize and depseudonymize data on Dapla.
...
 
 # Example: Single field sid mapping and pseudonymization (FPE)
 result_df = (
-    PseudoData.from_pandas(df)                     # Specify what dataframe to use
-    .on_field("fnr")                               # Select the field to pseudonymize
-    .map_to_stable_id()                            # Map the selected field to stable id
+    Pseudonymize.from_polars(df)                   # Specify what dataframe to use
+    .on_fields("fnr")                              # Select the field to pseudonymize
+    .with_stable_id()                              # Map the selected field to stable id
     .pseudonymize()                                # Apply pseudonymization to the selected fields
```
Comment (bjornandre) [inline README.md:65]:
```suggestion
    .run()                                # Apply pseudonymization to the selected fields
```

**Comment by bjornandre** [comment]
Good work! Just a few minor suggestions

**Comment by mallport** [comment]
Good catch @bjornandre! 😄

**Review by bjornandre — APPROVED** [review]

**Pull request merged**